sources/js/utils/rb_storage.js

```diff
@@ -16,7 +16,13 @@
     rb.storage = {};
 
     ['session', 'local'].forEach(function(type){
-        var storage = window[type + 'Storage'];
+        var storage = undefined;
```

Comment:
all var should be at the start of the function. You don't need to declare a variable by explicitly assigning undefined.

Should look something like this:

```
        var storage;
        var testStr = rb.getID();

        try{
            storage = window[type + 'Storage'];
        } catch(e) {
            storage = {};
        }
```